doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1055,32 +1031,37 @@ of interpretations within a scope).
 
 .. cmd:: Open Scope @scope
 
-   The command to add a scope to the interpretation scope stack is
-   :n:`Open Scope @scope`.
+   Adds a scope to the interpretation scope stack.
+
+   When the command appears in a section, the scope is only added within the section.
+   When the command is outside a section, the scope is exported as part of the current module.
+
+   This command supports the :attr:`local` and :attr:`global` attributes.
+   :attr:`local` adds the scope only in the current module.  :attr:`global`
+   adds the scope outside the current section but has no effect if the
```

Comment:
Not just outside the current section but outside all sections, in the innermost enclosing module.  Does this nuance matter?

Reply:
How is this?  We still need further work on the descriptions of `local` and `global`; we should document the various semantic patterns very clearly and have the commands refer to them.  Repeating similar somewhat nontrivial wording everywhere is harder to read.

```
   :attr:`global`
   adds the scope outside all currently open sections in the innermost open
   module.  :attr:`global` has no effect if the command is not inside a section.
```

Reply:
That seems good to me